orc8r/cloud/go/services/metricsd/prometheus/prometheus-cache/cache/cache_test.go

```diff
@@ -92,15 +103,25 @@ func TestScrape(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusOK, rec.Code)
 
+	// since families are processed in an arbitrary order
+	// we can't check the string exactly. we can check
+	// how many lines it has. the rest of the tests check
+	// whether the metrics are sorted properly. there are 4 extra
+	// debug statements but 1 less newline in the output.
+	assert.Equal(t, strings.Count(sampleReceiveString, "\n")+4-1, strings.Count(rec.Body.String(), "\n"))
```

Comment:
does this check matter if we're parsing and verifying the output string later?

Reply:
The difference is that the parsing that happens later counts the # of families, whereas this tries to make sure we didn't lose any metrics. We can check for metrics later too by doing a nested loop on each family, but this essentially accomplishes the same thing. 

I think its probably cleaner to do it after parsing, so I'll make that change.